### DISCUSSION TITLE
Headings font size

### DISCUSSION THREAD

**Discussion by kravorkid** [quasarframework/quasar#15024] [Ideas / Proposals] [answered]
Hi ! i would like to know if it's possible to rework headings default font size and spacing i think they are way too big. Every time i create a quasar project i need to override those variables to get a decent font size & spacing maybe you could copy the [default one from bootstrap ](https://getbootstrap.com/docs/5.2/content/typography/) ?

**Comment by metalsadman** [comment, marked as answer]
it would introduce a breaking change, what you could do is make a class of your own, then re-use that in your app. if you mean the <h> tags, those are native and quasar doesn't style those.

**Reply by kravorkid** [reply]
Thanks for you fast answer that's what i've done so far. I didn't think it was a breaking thing

**Reply by metalsadman** [reply]
it would for peeps that are already using the current typography classes.

**Reply by metalsadman** [reply]
might be considered for next version though, as I agree they are quite bigger than usual.

**Comment by Flamenco** [comment]
LOL the default heading fonts are GIGANTIC.

It would not be breaking if you simply added a configuration option, such as:

```
config: {
   useReasonableHeadingFontSize: true
}
```

Then sometime down the road, after deprecation warning, swapped it with
config: {
   useOldStyleGiganticHeadingFontSize: true
}
```

**Comment by ivanjaros** [comment]
actually you can just do this in your **quasar.variables.scss** file
```
$h1: (size: 1.7rem,   line-height: 1.7rem,  weight: 300) !default;
$h2: (size: 1.55rem,  line-height: 1.55rem, weight: 300) !default;
$h3: (size: 1.45rem,  line-height: 1.45rem, weight: 400) !default;
$h4: (size: 1.2rem,   line-height: 1.2rem,  weight: 400) !default;
$h5: (size: 1rem,     line-height: 1rem,    weight: 400) !default;
$h6: (size: 0.75rem,  line-height: 0.75rem, weight: 500) !default;
```

**Comment by Flamenco** [comment]
@ivanjaros  The issue is not how to change the values, it's that the default values are not practical.

**Reply by ivanjaros** [reply]
oh, yeah. that is 100% true. simple official fix, with backwards compatibility, would be to do the reverse. set defaults to some sane values and then use what i posted as default fix. people can then simply remove these overrides and that's it.

**Reply by ivanjaros** [reply]
or it could even be part of the installation process. like picking vite vs webpack.

**Comment by ivanjaros** [comment]
I would also add request for button sizes. The md should be more like sm.
